src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.cs

```diff
@@ -57,6 +57,20 @@ private static bool TryGetChallengeDataForScheme(string scheme, HttpHeaderValueC
             return false;
         }
 
+        internal static bool IsSessionAuthenticationChallenge(HttpResponseMessage response)
+        {
+            HttpHeaderValueCollection<AuthenticationHeaderValue> authenticationHeaderValues = GetResponseAuthenticationHeaderValues(response, false);
```

Comment:
Nit: please name the bool argument; otherwise it's hard to tell when reading the code what "false" means.